CHANGELOG.md

```diff
@@ -32,42 +32,42 @@ This project adheres to [Semantic Versioning](https://semver.org/).
    * Added missing unit tests for `pysat.utils.time`
    * Added missing unit tests for `pysat.utils.file.parse_delimited_filename`
 
-[3.0.1] - 2021-XX-XX
---------------------
+## [3.0.1] - 2021-07-28 
 * New Features
-   * Added a routine for loading CSV files into a pandas DataFrame from a list
-     of filenames.
-   * Added check for supported `tag` and `inst_id` at pysat.Instrument
-     instantiation. (#784)
-   * Expanded Constellation utility by:
-      * Adding common properties: `empty`, `index`, `date`, `today`,
-        `yesterday`, `tomorrow`, and `variables` (#764)
-      * Improving the printed output to inform user of the Constellation
-        contents (#764)
-      * Added methods to download data and create a common time index. (#764)
-   * Added utils.listify, a function that returns a list of whatever is input.
-     (#766)
-    * Added a warning for download requests that result in an empty date range.
+  * Added a routine for loading CSV files into a pandas DataFrame from a list
+    of filenames.
+  * Added check for supported `tag` and `inst_id` at pysat.Instrument
+    instantiation. (#784)
+  * Expanded Constellation utility by:
+    * Adding common properties: `empty`, `index`, `date`, `today`, `yesterday`,
+      `tomorrow`, and `variables` (#764)
+    * Improving the printed output to inform user of the Constellation contents
+      (#764)
+    * Added methods to download data and create a common time index. (#764)
+  * Added utils.listify, a function that returns a list of whatever is input.
+    (#766)
+  * Added a warning for download requests that result in an empty date range.
```

Comment:
Most of this is more correct on develop than it is in the RC.  The heading style and spacing needs to be adjusted to remove most of these differences.